PocketKit/Sources/PocketKit/MyList/SearchItemsList/SearchView.swift

```diff
@@ -13,6 +13,7 @@ struct SearchView: View {
         switch viewModel.searchState {
         case .emptyState(let emptyStateViewModel):
             SearchEmptyView(viewModel: emptyStateViewModel)
+                .environmentObject(viewModel)
```

Comment:
Read up on the benefits of environment objects https://www.hackingwithswift.com/quick-start/swiftui/what-is-the-environmentobject-property-wrapper

pretty cool

Reply:
Pretty cool indeed! Comes with a couple of caveats (making sure the object exists in the environment, otherwise it'll crash, and has some limitation if you want to define more than one `environmentObject` of the same type), but it's an elegant way to inject dependencies at inner scopes.